src/chttpd/src/chttpd_db.erl

```diff
@@ -1696,6 +1698,15 @@ extract_header_rev(Req, ExplicitRev) ->
         throw({bad_request, "Document rev and etag have different values"})
     end.
 
+validate_security_can_be_edited(DbName) ->
+    UserDbName = config:get("chttpd_auth", "authentication_db", "_users"),
+    CanEditUserSecurityObject = config:get("couchdb","users_db_security_editable","false"),
+    case {DbName,CanEditUserSecurityObject} of
+        {UserDbName,"false"} ->
+            Msg = "You can't edit the security object of the user database.",
```

Comment:
Normally the error message is something like this: not_found
Should I replace with a similar message? Do you have any suggestions?

Reply:
this is fine, `not_found` here is the `forbidden` in the next line. The message is fine.